<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QMainWindow" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="img_area">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1240</width>
      <height>500</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1238</width>
       <height>498</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="bottom_info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>1240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x, y</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menupoisson">
    <property name="title">
     <string>switch</string>
    </property>
    <addaction name="actiongo_to_poisson"/>
   </widget>
   <addaction name="menupoisson"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiongo_to_poisson">
   <property name="text">
    <string>go to poisson</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
